content/en/hosting/costs.md

```diff
@@ -25,9 +25,9 @@ In a production environment, the CHT costs USD$0.10 per active user per month to
 
 Medic hosts a number of production CHT instances in Amazon [Elastic Kubernetes Service](https://aws.amazon.com/eks/) (EKS). By using [OpenCost](https://www.opencost.io/), Medic can closely monitor real world costs with actual end users. OpenCost was used to separate one deployment's CPU, RAM and disk use from another's despite them all running in a multi-tenant EKS cluster.
 
-By looking at dozens of instances and focusing on the highest usage ones, an accurate representation can be derived of a higher cost a self hosting partner.  
+By looking at dozens of instances and focusing on the highest usage ones, hosting partners can know our numbers are applicable.  
```

Comment:
nitpick: I am unsure what this sentence means, to be honest. 😅 Is there a way to use simpler language, like: The numbers below are backed by the existing CHT deployments.

Reply:
you and Diana were both confused by this - let's yank it! surrounding text is clarifying enough.  see 6550fd90